### PR TITLE
fix: list items name (context page)

### PR DESCRIPTION
**Pull request by douscriptist**

### CODE REVIEW

File: content/docs/context.md [hunk 1]
```diff
@@ -9,20 +9,20 @@ Context, prop'ları her seviyede manuel olarak geçmek zorunda kalmadan bileşen
 Tipik bir React uygulamasında veri prop'lar aracılığıyla yukarıdan aşağıya aktarılır (üst bileşenlerden alt bileşenlere), fakat bu bir uygulamada birçok bileşene ihtiyaç duyulan belirli tipteki prop'lar (örneğin; lokalizasyon, arayüz teması) için kullanışsız olabilir. Context, bileşen ağacın her bir seviyesi üzerinden açıkça bir prop geçirmeden, bileşenler arasında bu gibi değerleri paylaşmanın bir yolunu sağlar.
 
 
-- [Context Ne Zaman Kullanılır {#when-to-use-context}](#context-ne-zaman-kullan%c4%b1l%c4%b1r-when-to-use-context)
-- [Context Kullanmadan Önce {#before-you-use-context}](#context-kullanmadan-%c3%96nce-before-you-use-context)
```
Comment (alioguzhan) [inline content/docs/context.md:13]: `[Context Kullanmadan Önce](#before-you-use-context)`

File: content/docs/context.md [hunk 2]
```diff
@@ -9,20 +9,20 @@ Context, prop'ları her seviyede manuel olarak geçmek zorunda kalmadan bileşen
 Tipik bir React uygulamasında veri prop'lar aracılığıyla yukarıdan aşağıya aktarılır (üst bileşenlerden alt bileşenlere), fakat bu bir uygulamada birçok bileşene ihtiyaç duyulan belirli tipteki prop'lar (örneğin; lokalizasyon, arayüz teması) için kullanışsız olabilir. Context, bileşen ağacın her bir seviyesi üzerinden açıkça bir prop geçirmeden, bileşenler arasında bu gibi değerleri paylaşmanın bir yolunu sağlar.
 
 
-- [Context Ne Zaman Kullanılır {#when-to-use-context}](#context-ne-zaman-kullan%c4%b1l%c4%b1r-when-to-use-context)
```
Comment (alioguzhan) [inline content/docs/context.md:12]: `[Context Ne Zaman Kullanılır](#when-to-use-context)`

**Review by alioguzhan — CHANGES_REQUESTED** [review]
Thanks for the fixes. But it is not working, still. Can you please remove the Turkish words from the links? Also, there are unnecessary English anchors like `{#xxx}`, we can remove those, too.

I left a few comments for example purposes.

File: content/docs/context.md (hunk 2 above)
Comment (douscriptist) [reply content/docs/context.md:12]: I did. Hope, did not miss any.
Also links may be broken because of unnecessary marks (``) removed them also.

**Comment by alioguzhan** [comment]
Thanks for the fixes ! @douscriptist Merging it...

**Pull request merged**